### PR TITLE
Feature/disabled banned

### PR DESCRIPTION
**Pull request by Schnoop**
https://github.com/FrankProjects/UltimateWarfare/issues/50

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -6230,6 +6230,73 @@
         }
     ],
     "packages-dev": [
+        {
+            "name": "masterminds/html5",
```
Comment (frank9999) [inline composer.lock:6234]: A lot of dependencies are added in composer.lock, but not in the composer file. Also these dependencies are unrelated to your actual implementation, please revert

**Review by frank9999 — COMMENTED** [review]
Thanks for your contribution. MR seems okay except for the composer.lock. In the future it would be an option to merge the banned/disabled functionality into the admin user list, with filter options. So for example you can filter the list by banned users, disabled users, etc

File: composer.lock (hunk 1 above)
Comment (Schnoop) [reply composer.lock]: Done.

**Review by frank9999 — APPROVED** [review]
Looks good!
